packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte

```diff
@@ -312,16 +314,16 @@
                 <tr class="table-row item-row">
                   <th scope="row" class="item-description">
                     <p class="truncate description-text" style="margin: 4px 0;">
-                      {item.name}
+                      {item.name || "-"}
                     </p>
                   </th>
-                  <td>{item.quantity}</td>
+                  <td>{item.quantity || "-"}</td>
                   <td
                     >{formatUnits(item.unitPrice, currency?.decimals ?? 18)}</td
                   >
                   <td>{formatUnits(item.discount, currency?.decimals ?? 18)}</td
                   >
```

Comment:
Do you need `|| "-"` on `item.unitPrice` and `item.discount` too?

Also, what's up with the formatting on these two lines?